packages/grafana-data/src/types/pluginExtensions.ts

```diff
@@ -204,7 +204,7 @@ export type PluginExtensionPanelContext = {
 export type PluginExtensionQueryEditorRowAdaptiveTelemetryV1Context = {
   /** An ordered list of lower-case [a-z]+ string identifiers to provide context clues of where this component is being embedded and how we might want to consider displaying it */
   contextHints?: string[];
-  query?: DataQuery;
+  query?: DataQuery & { expr?: string };
```

Comment:
In the PR description it says _"(...) PluginExtensionQueryEditorRowAdaptiveTelemetryV1Context is expected to have an expr."_ - is it ok that it's an optional?

(In case it should be mandatory, then we could also introduce a new version I think?
```ts
export type PluginExtensionQueryEditorRowAdaptiveTelemetryV2Context = {
  contextHints?: string[];
  query?: DataQuery & { expr: string };
}
```